toolkit/docs/building/building.md

```diff
@@ -803,6 +803,7 @@ To reproduce an ISO build, run the same make invocation as before, but set:
 | ALLOW_TOOLCHAIN_REBUILDS      | n                                                                                                      | Do not treat rebuilds of toolchain packages during regular package build phase as errors.
 |  PACKAGE_BUILD_RETRIES        | 1                                                                                                      | Number of build retries for each package
 | CHECK_BUILD_RETRIES           | 1                                                                                                      | Minimum number of check section retries for each package if RUN_CHECK=y and tests fail.
+| EXTRA_BUILD_LAYERS            | 0                                                                                                      | How many additional layers of the build graph to build beyond the requested packages (useful for testing changes in dependent packages)
```

Comment:
Won't the value of 0 change the current default behaviour where we're rebuilding everything, that depends on an updated package?

Reply:
This is how many additional layers we should add, we always have an implicit layer 0 which includes the packages that are directly requested.